testing/test_package_docs/index.html

```diff
@@ -84,11 +84,11 @@ <h1 class="title">
 <p>This is an amazing package.</p>
 <p>For example, it:</p><ul><li>Is very fast</li><li>Has zero bugs</li><li>Is free</li></ul>
 <p>It also has some awesome code</p>
-<pre class="prettyprint language-dart"><code class="language-dart">void main() {
+<pre class="dart prettyprint"><code>void main() {
```

Comment:
Not sure why this changed?

I did re-gen the docs (`` pub global run grinder update-test-package-docs``) but I don't see how my change could have affected this bit.  Maybe this is left over from another change?